website/content/api-docs/auth/kubernetes.mdx

```diff
@@ -65,11 +67,10 @@ See [these instructions](/vault/docs/auth/kubernetes#discovering-the-service-acc
 If Vault is running in a Kubernetes Pod, the `kubernetes_ca_cert` and
 `token_reviewer_jwt` parameters will automatically default to the local CA cert
 (`/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`) and local service
-account JWT (`/var/run/secrets/kubernetes.io/serviceaccount/token`). This
-behavior may be disabled by setting `disable_local_ca_jwt` to `true`.
+account JWT (`/var/run/secrets/kubernetes.io/serviceaccount/token`).
 
-When Vault is running in a non-Kubernetes environment, either
-`kubernetes_ca_cert` or `pem_keys` must be set by the user.
+If this behavior is disabled by setting `disable_local_ca_jwt` to `true`,
```

Comment:
Which behavior are you referring to?

Reply:
I'm referring to the behavior of `kubernetes_ca_cert` defaulting  to the local CA cert if Vault is running a Kubernetes pod from the previous paragraph https://github.com/hashicorp/vault/blob/77142a986adbcd634b77e985a4b34486a5f14824/website/content/api-docs/auth/kubernetes.mdx?plain=1#L67-L70

I can reiterate to make it clearer here. Let me know your thought.
```suggestion
If `kubernetes_ca_cert` defaulting to the local CA cert is disabled by setting `disable_local_ca_jwt` to `true`,
```